### ISSUE TITLE
The problem with Premade vs Premade priority

### ISSUE THREAD

**Issue by nacsdale** [LightsHope/issues#776]
Over the past few weeks i've been talking to numerous people that actively PvP on the server and we've come to a conclusion that pre vs pre priority ruins the game. The queue system as of now marks you as a premade if you're queued with more than 4 people and will ignore the queue if there is a group of 4+ queued aswell on the opposite faction. There are numerous problems with this system and i'll try to explain why. Right now the horde rankers are forced to soloqueue or play at night due to how the system works. If you premade you'll get thrown in a pre vs pre game everytime due to the amount of premades on alliance. This way you eat up all the prem games and you allow pug vs pug games to open up meaning the soloqueuers will outfarm you honor wise due to the fact that you'll be stuck in 20-30 games that only award around 1500 honor on a non weekend if we talking ab cause of DR on honorable kills (killin same ppl every game etc). The current system encourages dodging the premades and afking in soloqueue games. If I am not mistaken this was not a thing on Lightshope as I ranked there myself on horde side. Even the less skilled people would premade cause it was better honor than soloqueuing because there was no priority. You would get an occasional premade vs premade game every 3/4 games. Basically as of now if you're horde you're forced to soloqueue and if you're alliance you just premade and 5cap every single game. I think that this is a really important issue and should be looked at by the dev team.

**Comment by fitigf15** [comment]
Cadet / Kadet here, ranked multiple times on Lightbringer and currently ranking on Northdale. Completely agreeing with nacsdale's point. Most of the highest RP brackets are currently filled with either dodgers or account sharers, which in my opinion is kinda sad. Premades on both sides are sitting on 15+ minute queues on prime time while pugs (including 4 man grps) get instant queues. The amount of premades has been drastically decreasing over time due to the honor inefficiency compared to solo queue. This game is made to encourage the players to team up, not to split apart. Premades are (or were) a big part of the active community, for both either casual or hardcore players.

**Comment by returnflag** [comment]
I feel that the current system in place is attempting to make BG's more noob friendly, as who want's to get rolled by a full premade? 
But when you have a system like this, you're effectively killing the premade scene, because who wants to get rolled by a rival premade team all the time?  who wants to sit in a long queue? this then leads to queue dodging, and a long q pop like Cadet says.

If you were to keep the current system (pls no), a good improvement would be where if you queued, your faction wouldn't matter. e.g. APES v Supremacy. this would lead to a greater pool of teams to face.

**Comment by Illianemploy** [comment]
I feel that the current system in place is attempting to make BG's more noob friendly, as who want's to get rolled by a full premade?
But when you have a system like this, you're effectively killing the premade scene, because who wants to get rolled by the same premade team all the time? who wants to sit in a long queue? this then leads to queue dodging, less premade teams queueing (ref progress) and a longer q pop like Cadet says.

If you were to keep the current system (pls no), a good improvement would be where if you queued, your faction wouldn't matter. e.g. APES v Supremacy. this would lead to a greater pool of teams to face.

**Comment by snawfu** [comment]
Best possible way to implement the queue system is Blizzlike aka first come first serve (no multiqueue) and deserter on dropping queue so opening games isn't possible.
Premade v premade prio ruined Nostalrius for a long time and the unwillingness to learn from past mistakes is bad for the server.

**Comment by nacsdale** [comment]
What Snawfu said. You'd think that the dev team would know better this time around after what happened on Nost and their attempt to remove premades back on Anathema and how it completely failed (this is not in any way an attempt to insult the stuff, all im saying is maybe after numerous questionable decisions you'd know better to just leave the queue system blizzlike)

for ref. https://i.gyazo.com/0da200d1b84fd3d7b12429277e1b685c.png

**Comment by tribe1234** [comment]
I agree that this is a problem, however I can confirm that the same queue system was in on lightbringer.

It's only a problem for whichever side gets farmed in premade vs premade, but there's a lot of issues with the system regardless.

**Comment by knasen88** [comment]
You do open up pugs to get absolutely demolished by premades(they should learn to fucking group up though), however current system just ENCOURAGES account sharing and grinding out honor 24/7. If you wanna keep current system atleast ban the people that's online for 23 hours a day because no human does that for more than 1 week, and that is ON meth.

**Comment by Slicy** [comment]
This : https://forum.nostalrius.org/viewtopic.php?f=46&t=30232
And this : "Re: Removal of premade vs. premade priority
by Nano » Tue Mar 22, 2016 5:58 pm
It was removed because the mechanic that was implemented did not prevent widespread abuse of the honor system like it was intended. Currently the queue system is the exact same as it was during vanilla retail and is unlikely to be changed again.
If you'd like to pursue more winning games, perhaps you should find some friends and organize yourselves together to queue."

Or this : https://forum.nostalrius.org/viewtopic.php?f=5&t=35301&hilit=premade+queue 

No need to explain in more details why the current system is and will be exploited. I've already been informed there is some shady wintrading going on between a horde and an ally prem. You don't want me on that crusade again.
